sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1859,22 +1871,30 @@ private synchronized void showDownloadProgress() {
      * Show modal dialog with install button if mandatory update ready and user cancelled install.
      */
     private synchronized void showMandatoryDownloadReadyDialog() {
-        if (shouldRefreshDialog(mCompletedDownloadDialog)) {
-            final ReleaseDetails releaseDetails = mReleaseDetails;
-            AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(mForegroundActivity);
-            dialogBuilder.setCancelable(false);
-            dialogBuilder.setTitle(R.string.appcenter_distribute_install_ready_title);
-            dialogBuilder.setMessage(getInstallReadyMessage());
-            dialogBuilder.setPositiveButton(R.string.appcenter_distribute_install, new DialogInterface.OnClickListener() {
 
-                @Override
-                public void onClick(DialogInterface dialog, int which) {
-                    installMandatoryUpdate(releaseDetails);
-                }
-            });
-            mCompletedDownloadDialog = dialogBuilder.create();
-            showAndRememberDialogActivity(mCompletedDownloadDialog);
+        /* Check if we are in foreground. */
```

Comment:
```suggestion
        /* Check if we are in background. */
```

Reply:
Nope. I'd rather change it to `Verify we are in foreground` (both comments)

Reply:
Agree with Dmitriy, I think that `background` will be more correct.

Reply:
Nope.
It's either:
Confirm\check\verify we are **not** in the background
Or
Confirm\check\verify we are in the foreground

Reply:
Or at least, something like "Return if we are in the background\not in the foreground" at least, as a compromise :D

Reply:
I'll adjust message then